pkg/entity/flag_snapshot.go

```diff
@@ -66,4 +68,21 @@ func SaveFlagSnapshot(db *gorm.DB, flagID uint, updatedBy string) {
 	if err := tx.Commit().Error; err != nil {
 		tx.Rollback()
 	}
+
+	logFlagSnapshotUpdate(flagID, updatedBy)
+}
+
+var logFlagSnapshotUpdate = func(flagID uint, updatedBy string) {
+	if config.Global.StatsdClient == nil {
+		return
+	}
+
+	config.Global.StatsdClient.Incr(
+		"flag.snapshot.updated",
```

Comment:
This will be namespaced by `flagr` right?

Reply:
yes